spec/lib/migrations/load_rate_reference_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+require 'rake'
+require 'roo'
+
```

Comment:
So for a few reasons this isn't a very successful test. I'll mark them at the lines. 

One thing to consider throughout - you're duplicating behavior on both sides: Roo in the task, Roo in the test, etc.

In a few cases this ends up hurting you because your test implicitly assumes something about how values should be parsed. 

Whenever you're writing a test you have to first think about WHAT it is that we seek to test. In this particular case we're going to write an integration test so we don't want to mock any external dependencies. 

If there were internal logic in the rake task - i,e you're converting/parsing values in some way, then it would probably help to have a unit test where we mocked Roo's response. 

So really for this test we want to run the importer against the xlsx file, and then check the ACTUAL created records that they match our expectations.